### DISCUSSION TITLE
How can I get "Mini" icons from @heroicons v2?

### DISCUSSION THREAD

**Discussion by johackim** [tailwindlabs/heroicons#865] [Help]
Hello,

Thanks for your icon library!

How can I get "Mini" icons?

There aren't inside @heroicons/react : https://unpkg.com/browse/@heroicons/react@2.0.12/

**Comment by ivan-kleshnin** [comment]
They are under `20/solid`:

```
import {MapPinIcon} from "@heroicons/react/20/solid"
```

**Comment by johackim** [comment]
Thanks @ivan-kleshnin!
